src/upstream.ts

```diff
@@ -49,7 +51,37 @@ export {
   V1TokenReview as TokenReview,
   V1ValidatingWebhookConfiguration as ValidatingWebhookConfiguration,
   V1VolumeAttachment as VolumeAttachment,
-  V1Endpoint as Endpoint,
+  // V1Endpoint as Endpoint, - keep this so we do not forget incase it is corrected
 } from "@kubernetes/client-node";
 
 export { GenericKind } from "./types";
+
+export class Endpoint {
```

Comment:
Can we test this under `kinds.test.ts`? It looks we have some sort of basic test for all the other kinds we support

Reply:
good idea

Reply:
ty!